tests/core/custom-device-reg-extras/task.yaml

```diff
@@ -37,7 +35,8 @@ prepare: |
     prepare_testrootorg_store
 
     # start fake device svc
-    systemd_create_and_start_unit fakedevicesvc "$(command -v fakedevicesvc) localhost:11029"
```

Comment:
Can we remove this helper "systemd_create_and_start_unit " now or is it still used?

Reply:
It is still used, there will be a second part for the migration and then we will be able to remove it